src/containerapp/azext_containerapp/custom.py

```diff
@@ -3656,6 +3660,72 @@ def disable_dapr(cmd, name, resource_group_name, no_wait=False):
     except Exception as e:
         handle_raw_exception(e)
 
+def init_dapr_component(cmd, resource_group_name, environment_name):
```

Comment:
What should we do if the Redis service already exists or there are already components created with the names we want to use? Wonder if we should ask for confirmation before proceeding (not sure what the CLI guidelines are for this, might need to add a `--yes` flag or something to bypass).

Reply:
![image](https://github.com/Azure/azure-cli-extensions/assets/22132944/c208cba9-d62c-4c2f-866d-9b895fe97f46)